jquery.scrolly.js

```diff
@@ -40,16 +40,11 @@
     this.velocity = this.$element.attr('data-velocity');
     this.bgStart = parseInt(this.$element.attr('data-fit'), 10);
 
-    $(document).scroll(function(){
+   $(document).bind('scroll', function(){
```

Comment:
I guess you're right!
As seen in http://stackoverflow.com/questions/11847021/jquery-bind-vs-on, bind maps to on, but will be deprecated!